From cf510e4aee93bf1392df8e35931ba9bb9a07fc16 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Wed, 25 Apr 2018 22:05:50 +0200
Subject: [PATCH] Fix build with Qt 5.11 (missing QDoubleValidator)

Reviewers: mutlaqja

Subscribers: #kde_edu

Tags: #kde_edu

Differential Revision: https://phabricator.kde.org/D12525
---
 kstars/dialogs/focusdialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kstars/dialogs/focusdialog.cpp b/kstars/dialogs/focusdialog.cpp
index d09ce9273..b264ee5c9 100644
--- a/kstars/dialogs/focusdialog.cpp
+++ b/kstars/dialogs/focusdialog.cpp
@@ -24,6 +24,7 @@
 #include "skyobjects/skypoint.h"
 
 #include <KLocalizedString>
+#include <QDoubleValidator>
 #include <KMessageBox>
 #include <QPushButton>
 
-- 
2.17.0

